bazel/BUILD

```diff
@@ -81,6 +81,7 @@ cc_library(
         "ngraph_bridge/ngraph_enter_prefetch_in_catalog.cc",
         "ngraph_bridge/ngraph_executor.cc",
         "ngraph_bridge/ngraph_encapsulate_op.cc",
+        "ngraph_bridge/ngraph_find_replace_prefetchdataset.cc",
```

Comment:
This should be removed since we do not do the prefetch for grappler build and bazel uses grappler.

Reply:
prefetch can be done for any build. it is guarded by the env flag

Reply:
ok then I'll leave it here